Shared/Coordinators/SettingsCoordinator.swift

```diff
@@ -118,6 +121,11 @@ final class SettingsCoordinator: NavigationCoordinatable {
     func makeVideoPlayerSettings() -> VideoPlayerSettingsCoordinator {
         VideoPlayerSettingsCoordinator()
     }
+    
+    func makefilterDrawerButtonSelector(selectedButtonsBinding: Binding<[FilterDrawerActionButton]>) -> some View {
+        FilterDrawerButtonSelectorView(selectedButtonsBinding: selectedButtonsBinding)
```

Comment:
The file for this view must have been linked incorrectly, Xcode on my machine says that it is in your `Documents` directory.

Reply:
Just to confirm, this is the Shared/Coordinators/SettingsCoordinator.swift file? That's the only file that's referenced like that or is it all of them?

I can take a look at this and try re-pushing if it's only the one file. It's everything then god help me haha

Reply:
I *believe* this is referenced correctly now! That being said, I'm not sure how this got out of sync in the first place so... I'll await from confirmation that this looks correct.